### DISCUSSION TITLE
identity_cert in device table

### DISCUSSION THREAD

**Discussion by ashishmeher216** [micromdm/nanomdm#118] [Q&A] [answered]
![image](https://github.com/micromdm/nanomdm/assets/51280607/94c00652-021e-4be3-92f6-117afa7f3f8e)

Can anyone please help me understand which certificate is identity_cert? And from where does it come?

**Comment by jessepeterson** [comment, marked as answer]
Take a look at the "Device Identity certificate" section of https://micromdm.io/blog/certificates/
